### PR TITLE
Decode HTML in Clues

### PR DESCRIPTION
**Pull request by bdenney**
Partially addresses #220 . Decoding HTML encoded clue values. Very unsure if this is the _best_ way to fix this, but it does appear to work 😄 

The puzzle provided in #220 does have a few examples of this but it doesn't appear I can upload that puz file to the issue 😢

### CODE REVIEW

File: src/components/Player/ClueText.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 import React from 'react';
 
+function decodeHtml(htmlText) {
```
Comment (stevenhao) [inline src/components/Player/ClueText.js:3]: any reason not to use this? https://lodash.com/docs/4.17.15#unescape

Comment (bdenney) [reply src/components/Player/ClueText.js:3]: I tried that and it wouldn't unescape the bulk of anything in the test puzzle mentioned in #220. I found this explanation: https://stackoverflow.com/questions/36352155/unescape-function-from-underscore-js-not-working

Comment (bdenney) [reply src/components/Player/ClueText.js:3]: https://github.com/mathiasbynens/he would also be an option, but not sure whether bringing a library in for this is appropriate. (As in genuinely don't know, pretty new to JS development 😄 )

Comment (stevenhao) [reply src/components/Player/ClueText.js:3]: gotcha! i think importing he or doing it as you did are both reasonable! thanks for the explanation

**Pull request merged**